.github/workflows/test-workflow.yml

```diff
@@ -8,10 +8,10 @@ on:
         type: string
 
 env:
-  RELAY_IMAGE_TAG: 0.23.3
-  NETWORK_NODE_IMAGE_TAG: 0.39.0-alpha.0
-  HAVEGED_IMAGE_TAG: 0.39.0-alpha.0
-  MIRROR_IMAGE_TAG: 0.81.0-rc1
+  RELAY_IMAGE_TAG: 0.29.1
+  NETWORK_NODE_IMAGE_TAG: 0.40.1
```

Comment:
```suggestion
  NETWORK_NODE_IMAGE_TAG: 0.41.4
```

Reply:
Doesn't look like the change was pushed.

Reply:
Yes, the package.json file was not staged.  It's been committed and pushed.  Apologies.

Reply:
I had to update to the following versions to get the local node to start:
`RELAY_IMAGE_TAG: 0.31.1`
`NETWORK_NODE_IMAGE_TAG: 0.42.0-alpha.0`
`HAVEGED_IMAGE_TAG: 0.42.0-alpha.0`
`MIRROR_IMAGE_TAG: 0.88.0`